src/Semantics.js

```diff
@@ -413,7 +413,7 @@ Semantics.prototype.addOperationOrAttribute = function(type, signature, actionDi
       return '[' + name + ' operation]';
     };
   } else {
-    Object.defineProperty(this.Wrapper.prototype, name, {get: doIt});
+    Object.defineProperty(this.Wrapper.prototype, name, {get: doIt, configurable: true});
```

Comment:
Nit: Might be nice to break this across lines, and add a comment why it needs to be configurable. E.g.:
```js
Object.defineProperty(this.Wrapper.prototype, name, {
  get: doIt,
  configurable: true  // So the property can be deleted.
});
```